### PR TITLE
Updates to Rereco2016

### PR DESCRIPTION
**Pull request by DouglasBurns**
ElectronID has been updated.
PseudoTop dependency has been removed
Electron channel uses new Lumi json files + updates in their crab configs.
BTagSF now use ICHEP recommendations
Updated Muon Triggers
Updated a MET filter (Seems we may have more to add)
Add a git cms-merge-topic for running MET corrections 

This PR includes a lot of back and forth in the electron trigger. (i.e. between Ele32 unprescale and Ele27 partial prescale)
This includes updates in Globaltags, lumi jsons, triggers etc and so would be uncompatible with the ICHEP dataset. Perhaps this should for now be in a new branch of NTupleProduction?

### CODE REVIEW

File: data/JSON/LSfor_HLT_Ele27_WPTight_Gsf_withLowestSeed_L1_SingleIsoEG26.json [hunk 1]
```diff
@@ -0,0 +1,377 @@
+
```
Comment (kreczko) [inline data/JSON/LSfor_HLT_Ele27_WPTight_Gsf_withLowestSeed_L1_SingleIsoEG26.json]: This is not a proper JSON file

File: python/Jets_Setup_cff.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def setup_jets(process, cms, options, postfix="PFlow"):
     if usePrivateSQlite:
         from CondCore.DBCommon.CondDBSetup_cfi import *
         import os
-        era = "Fall15_25nsV2_"
+        era = "Spring16_25nsV6_"
```
Comment (kreczko) [inline python/Jets_Setup_cff.py]: A new era begins

File: python/ntp/commands/run/analysis/DICE.py [hunk 3]
```diff
@@ -208,4 +227,4 @@ def create_job_layer(self, input_files, mode):
             job_set.add_job(job)
             jobs.append(job)
 
-        return jobs
+        return jobs
```
Comment (kreczko) [inline python/ntp/commands/run/analysis/DICE.py]: New lines at the end of a file are also useful

File: python/ntp/commands/run/analysis/__init__.py [hunk 4]
```diff
@@ -57,6 +57,24 @@
 ]
 
 
+def input_files_from_path(path):
```
Comment (kreczko) [inline python/ntp/commands/run/analysis/__init__.py]: this should be available in `ntp.commands.run`. Why duplicate it?

File: python/ntp/commands/run/analysis/template.py [hunk 5]
```diff
@@ -163,7 +169,7 @@ def getAnalysisSettings(analysisMode):
 TQAFPath = ""
 
 # integrated luminosity the MC simulation will be scaled to
-lumi = 15933
+lumi = 12878
```
Comment (kreczko) [inline python/ntp/commands/run/analysis/template.py]: we have now less lumi?

File: python/userdata/JetUserData_cfi.py [hunk 6]
```diff
@@ -6,7 +6,7 @@
     jetCollection=cms.InputTag("patJetsReapplyJEC"),
     beamSpotCollection=cms.InputTag('offlineBeamSpot'),
     # Top Object Definitions
-    minSignalJetPt=cms.double(20.),
+    minSignalJetPt=cms.double(30.),
```
Comment (kreczko) [inline python/userdata/JetUserData_cfi.py]: Just need to keep this in mind. Affects HT distribution.

**Comment by DouglasBurns** [comment]
Removed JSON Electron JSON that wasnt JSON (dont need anyway if we are using Ele32)
Removed duplicated function
Addes Moriond golden JSON
Added a couple more DY datasets from current tranche (though old tune)
Update Lumi

**Pull request merged**